src/domain/alerts/alerts.repository.interface.ts

```diff
@@ -12,7 +12,8 @@ export interface IAlertsRepository {
 
   /**
    * Parses and notifies the user about the {@link log} from the Alerts provider
+   * @param chainId - chain where the alert log was generated
    * @param log - the {@link AlertLog} to decode
    */
-  handleAlertLog(log: AlertLog): void;
+  handleAlertLog(chainId: string, log: AlertLog): void;
```

Comment:
This should return `Promise<void>`.

Reply:
Changed in [65990f9](https://github.com/safe-global/safe-client-gateway/pull/872/commits/65990f9a4c21b6e17281eb61679349b56e5c9818)